File: app/src/main/java/com/softaai/designpatterns/java/creational/builder/meal/item/Burger/ChickenBurger.java
package com.softaai.designpatterns.java.creational.builder.meal.item.Burger;

/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
public class ChickenBurger extends Burger {
    @Override
    public String name() {
        return "Chicken Burger";
    }

    @Override
    public float price() {
        return 55.5f;
    }
}

File: app/src/main/java/com/softaai/designpatterns/kotlin/creational/builder/meal/item/Burger/ChickenBurger.kt
package com.softaai.designpatterns.kotlin.creational.builder.meal.item.Burger


/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
class ChickenBurger : Burger() {
    override fun price(): Float {
        return 55.50f
    }

    override fun name(): String {
        return "Chicken Burger"
    }
}